### PR TITLE
Enable multi-arch build in Makefile

### PR DESCRIPTION
**Pull request by jingxu97**
enable multi-arch build

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: test/k8s-integration/main.go [hunk 1]
```diff
@@ -331,6 +332,7 @@ func handle() error {
 		if err != nil {
 			return fmt.Errorf("failed to prepull images: %s, err: %v", out, err)
 		}
+		time.Sleep(15 * time.Minute)
```
Comment (msau42) [inline test/k8s-integration/main.go:335]: Is there a more deterministic way to detect when the prepull is complete? Is the script blocking?

Comment (jingxu97) [reply test/k8s-integration/main.go:335]: the script creates a daemonSet with desired image and the pods will download the container image. Similar to driver install and other windows tests, we currently just wait for sometime. We can improve this part by waiting for pod state changes next step.

Comment (msau42) [reply test/k8s-integration/main.go:335]: Please open an issue so we can track followup items

**Review by msau42 — COMMENTED** [review]
/lgtm

**Comment by jingxu97** [comment]
resolved the merge conflicts.PTAL @msau42 @mattcary

**Comment by jingxu97** [comment]
pull-gcp-compute-persistent-disk-csi-driver-e2e-win2019
pull-gcp-compute-persistent-disk-csi-driver-e2e-win1909

**Comment by jingxu97** [comment]
PTAL @msau42 @mattcary

**Pull request merged**